cnf-tests/testsuites/pkg/networks/network.go

```diff
@@ -30,43 +51,44 @@ func NewNetworkAttachmentDefinitionBuilder(namespace, nadName string) *NetworkAt
 	}
 }
 
-func (b *NetworkAttachmentDefinitionBuilder) WithAdditonalPlugin(config string) *NetworkAttachmentDefinitionBuilder {
-	b.configs = append(b.configs, config)
+func (b *NetworkAttachmentDefinitionBuilder) WithTuning(sysctls string) *NetworkAttachmentDefinitionBuilder {
+	b.metaPluginConfigs = append(b.metaPluginConfigs, sysctls)
 	return b
 }
 
-func (b *NetworkAttachmentDefinitionBuilder) WithTuning(sysctls string) *NetworkAttachmentDefinitionBuilder {
-	b.configs = append(b.configs, sysctls)
+func (b *NetworkAttachmentDefinitionBuilder) WithBond(bondName, link1, link2 string) *NetworkAttachmentDefinitionBuilder {
+	b.setConfig(fmt.Sprintf(bondConfig, bondName, link1, link2))
```

Comment:
I'd leave the format string explicit here, so the behaviour of this function is clearer (and the const is not reused anywhere)

Reply:
fixed